src/index.html

```diff
@@ -8,5 +8,39 @@
   </head>
   <body>
     <h1>Calendar</h1>
+
+    <div class="calendar calendar--start-day_fri calendar--month-length_28">
```

Comment:
```suggestion
    <div class="calendar calendar--fri calendar--28">
```
Можно классы вот так записать